# HG changeset patch
# User Rik <rik@octave.org>
# Date 1457670405 28800
#      Thu Mar 10 20:26:45 2016 -0800
# Node ID 10c32f940636e893920be82b285a5fb3b7cf2fba
# Parent  1da428cbf401573ef60c3fa74cc0bbec47934c90
common.mk: Add comment that SHLLINKEXT variable may be unnececssary.

* common.mk: Add comment that SHLLINKEXT variable may be unnececssary.

diff --git a/build-aux/common.mk b/build-aux/common.mk
--- a/build-aux/common.mk
+++ b/build-aux/common.mk
@@ -9,17 +9,19 @@ version := ${OCTAVE_VERSION}
 api_version := ${OCTAVE_API_VERSION}
 
 ## AM_LIBTOOLFLAGS = --silent
 
 AM_LFLAGS = @LFLAGS@
 
 AM_YFLAGS = -dv
 
-SHLLINKEXT=
+# FIXME: This seems unnecessary as of 3/10/2016.
+# Commenting out definition and re-configuring made no change to Makefile
+SHLLINKEXT =
 
 # Fortran compiler flags.
 
 AM_FFLAGS = @FFLAGS@
 
 # C compiler flags.
 
 AM_CFLAGS = ${XTRA_CFLAGS}
